_posts/2016-07-23-reopening-instructor-training.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+subheadline: "Two open instructor training classes this fall"
+title: "Reopening Instructor Training"
+teaser: ""
+header:
+   image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+   - blog
+comments: true
+show_meta: true
+authors: ["Greg Wilson"]
+---
+
+For the last ten months,
+the [Software Carpentry Foundation](http://software-carpentry.org/scf/) has worked toward three goals
+for its instructor training program:
+
+1.  Make the content more relevant.
+
+2.  Increase the number of people able to deliver instructor training.
+
+3.  Find a format that meets everyone's needs in a sustainable way.
+
+They have made a lot of progress on all three,
+and are therefore now able to offer instructor training once again
+to people who aren't affiliated with our partner organizations.
+If you wish to apply to take part
```

Comment:
Add something like:
"Prospective Software Carpentry and Data Carpentry instructors should complete the same instructor training class.  Prospective instructors will then complete a series of final tasks depending on whether they are seeking Software Carpentry certification, Data Carpentry Certification, or both."